camera/MultiCameraApplication/java/com/intel/multicamera/MultiViewActivity.java

```diff
@@ -142,7 +143,15 @@ protected void onCreate(Bundle savedInstanceState) {
                 public void onReceive(Context context, Intent intent) {
                     String action = intent.getAction();
                     if (UsbManager.ACTION_USB_DEVICE_ATTACHED.equals(action)) {
-                        System.out.println(TAG + "BroadcastReceiver USB Connected");
+                        UsbDevice usbDevice = intent.getParcelableExtra(UsbManager.EXTRA_DEVICE);
+                        if(usbDevice.getDeviceClass() == ic_camera.getUsb_dev_id()) {
+                            Log.d(TAG, "USB camera insert detected ");
+                        }
+                        else {
+                            Log.d(TAG, "Non USB camera device connected ");
```

Comment:
Change Non to No

registering of receiver inside broadcast receiver is not good. May I know why we are doing inside broadcast receiver and also outside broadcast receiver?